.drone.yml

```diff
@@ -37,6 +37,20 @@ pipeline:
   #   commands:
   #     - tests/wait_until_previous_builds_complete.sh
 
+  check-org-membership:
+    image: 'wdc-harbor-ci.eng.vmware.com/default-project/vic-integration-test:1.44'
+    pull: true
+    environment:
+      BIN: bin
+      GOPATH: /go
+      SHELL: /bin/bash
+    secrets:
+      - github_automation_api_key
+    commands:
+      - [[ ! $(curl --silent "https://api.github.com/orgs/vmware/members/${DRONE_COMMIT_AUTHOR}?access_token=${GITHUB_AUTOMATION_API_KEY}") ]]
```

Comment:
I think curly braces needs to be removed on drone variables. So just `$DRONE_COMMIT_AUTHOR`.